apps/codeforafrica/src/components/Footer/FooterDescription.js

```diff
@@ -1,19 +1,28 @@
 import { Link } from "@commons-ui/next";
 import { Typography } from "@mui/material";
-import Image from "next/image";
 import PropTypes from "prop-types";
 import React from "react";
 
-import cfaIcon from "@/codeforafrica/assets/images/CfAlogoBW.png";
+import Figure from "@/codeforafrica/components/Figure";
 
-function FooterDescription({ description }) {
+function FooterDescription({ description, logo }) {
   if (!description) {
     return null;
   }
   return (
     <>
       <Link href="/">
-        <Image src={cfaIcon} alt="" />
+        <Figure
+          priority
```

Comment:
```suggestion
```

Footer is low [`priority`](https://nextjs.org/docs/api-reference/next/image#priority)